src/coreclr/nativeaot/Runtime/unix/PalCreateDump.cpp

```diff
@@ -618,3 +619,8 @@ PalCreateDumpInitialize()
 
     return true;
 }
+#else
+void PalCreateCrashDumpIfEnabled()
+{
+}
+#endif // !defined(HOST_WASM)
```

Comment:
It would be cleaner to create a no-op `PalCreateDump.cpp` under the `/wasm` directory and so avoid modifying upstream code.

WASM does have the concept of a core dump, but it'll likely take years before we will be able to collect one from within WASM itself (or even JS).